examples/dataproc/main.py

```diff
@@ -19,10 +23,12 @@
     batch,
     max_age='1h',
     packages=['click', 'selenium'],
-    init=['gs://cpg-common-main/hail_dataproc/install_common.sh'],
-    cluster_name='My Cluster with max-age=1h',
+    init=[
+        'gs://cpg-common-main/hail_dataproc/2023-11-22-mfranklin-dev/install_common.sh'
```

Comment:
It's just an example, but are we going to update this back to something a little less bespoke…

Reply:
Yes absolutely!